origin.spec

```diff
@@ -311,6 +311,18 @@ install -m 0644 contrib/tuned/man/tuned-profiles-origin-node.7 %{buildroot}%{_ma
 
 mkdir -p %{buildroot}%{_sharedstatedir}/origin
 
+install -d %{buildroot}/usr/local/bin/
+
+# Install master scripts
+install -p -m 0755 images/origin/system-container/origin-system-container-wrapper.sh %{buildroot}/usr/local/bin/
```

Comment:
I guess I can change the rest of the scripts but for the `node` image there is an issue with /usr/bin and docker: https://github.com/openshift/origin/blob/master/images/node/scripts/docker#L14